Misc/ACKS

```diff
@@ -1886,3 +1886,4 @@ Aleksandr Balezin
 Robert Leenders
 Ngalim Siregar
 Tim Gates
+Karthikeyan Singaravelan
```

Comment:
You are already in the list.

Reply:
It was a result of the cherry pick. I will remove it here for conflicts with other PR sorting results. Thanks.